### PR TITLE
fix: use different image registry for busybox

### LINKED ISSUE #403: BUG: Don't use busybox image in SaaS
SaaS will block other registries to control what images are deployed.

We need to adjust the dataloss helper, to specific the [initcontainer ](https://github.com/zeebe-io/zeebe-chaos/blob/4dbdaff6ea8f2d44069aea164f50beca0effcab0/go-chaos/internal/dataloss_sim_helper.go#L67-L98) and use `gcr.io/camunda-saas-registry/busybox:1.36.1`

Related discussion https://camunda.slack.com/archives/CSQ2E3BT4/p1694600290842769?thread_ts=1694446510.955439&cid=CSQ2E3BT4

### PR DESCRIPTION
**Pull request by deepthidevaki**
Busybox image is set differently for SaaS and SM. For SM the tag is set to `latest`. But it seems tag latest is not available in SaaS registry. So it is not set to the latest version `1.36.1`. 

closes #403

### CODE REVIEW

**Review by rodrigo-lourenco-lopes — APPROVED** [review]
LGTM! 🚀 🚀

File: go-chaos/internal/dataloss_sim_helper_test.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Camunda Services GmbH
```
Comment (Zelldon) [inline go-chaos/internal/dataloss_sim_helper_test.go:1]: 🚀

**Review by Zelldon — APPROVED** [review]
Thank you! 🤗

File: go-chaos/internal/dataloss_sim_helper_test.go (hunk 1 above)
Comment (oleschoenburg) [reply go-chaos/internal/dataloss_sim_helper_test.go:1]: I guess that should be 2023 :sweat_smile:

**Pull request merged**